api/utils.go

```diff
@@ -130,3 +130,18 @@ func (api *API) validateAdminRequest(username string) error {
 	}
 	return nil
 }
+
+func (api *API) extractPostForms(formNames []string, c *gin.Context) map[string]string {
```

Comment:
idea: use spread operator:

```suggestion
func (api *API) extractPostForms(c *gin.Context, formNames ...string) map[string]string {
```